setup.cfg

```diff
@@ -61,6 +61,7 @@ where=src
 test =
     pytest
     pandas; python_version>"3.4"
+    tabulate
```

Comment:
I'd probably list this in dev, too?

Reply:
Makes sense. Just added.

Reply:
And all, just below, for sure!

Reply:
Oops, damn .. for sure! (I am swinging between 2 totally different bits of work, and that's not good ;-)) Will commit in a sec.